make/upgrade

```diff
@@ -34,18 +34,22 @@ helm_args=(
     --set "env.DOMAIN=${DOMAIN}"
     --set "env.TCP_DOMAIN=${TCP_DOMAIN}"
     --set "env.UAA_HOST=${UAA_HOST}"
+    --set "env.UAA_PORT=${UAA_PORT}"
     --set "env.INSECURE_DOCKER_REGISTRIES=${INSECURE_DOCKER_REGISTRIES}"
     --set "secrets.UAA_CA_CERT=${UAA_CA_CERT}"
     --set "kube.storage_class.persistent=${STORAGE_CLASS}"
 )
 
-# The extra IP address for `kube.external_ips` is to ensure that we can
-# create services binding to multiple addresses correctly.  It points at
-# an address that shouldn't exist in real life.
-helm_args+=(
-    --set "kube.external_ips[0]=192.0.2.42"
-    --set "kube.external_ips[1]=$(getent hosts "${DOMAIN}" | awk 'NR=1{print $1}')"
-)
+if [ -n "${INGRESS_CONTROLLER}" ]; then
+    helm_args+=(
+        --set "services.ingress.class=${INGRESS_CONTROLLER}"
+        --set "services.ingress.backends.router.port=443"
+    )
+else
+    helm_args+=(
+        --set "kube.external_ips[0]=$(getent hosts "${DOMAIN}" | awk 'NR=1{print $1}')"
```

Comment:
You're dropping the check on multiple IPs (see the comment that is being deleted).

Reply:
I saw the comment. Isn't this testing someone else's problem? We know kube's `externalIPs` work with multiple IP addresses.

Reply:
For the services, yes; but this is testing that we propagate it down correctly (because we had issues before where it didn't). It used to be called `kube.external_ip` or something, and only supported one.

Reply:
As discussed in person, Jenkins would catch any abnormal behavior. Leaving the code as is in the PR.